src/vs/workbench/api/electron-browser/mainThreadTerminalService.ts

```diff
@@ -55,14 +55,15 @@ export class MainThreadTerminalService implements MainThreadTerminalServiceShape
 		// when the extension host process goes down ?
 	}
 
-	public $createTerminal(name?: string, shellPath?: string, shellArgs?: string[], cwd?: string, env?: { [key: string]: string }, waitOnExit?: boolean): Thenable<number> {
+	public $createTerminal(name?: string, shellPath?: string, shellArgs?: string[], cwd?: string, noInheritEnv?: boolean, env?: { [key: string]: string }, waitOnExit?: boolean): Thenable<number> {
 		const shellLaunchConfig: IShellLaunchConfig = {
 			name,
 			executable: shellPath,
 			args: shellArgs,
 			cwd,
 			waitOnExit,
 			ignoreConfigurationCwd: true,
+			noInheritEnv: noInheritEnv,
```

Comment:
Nice shorthand in TS:

```suggestion
			noInheritEnv,
```